plugin-server/src/main/ingestion-queues/batch-processing/each-batch-webhooks.ts

```diff
@@ -145,7 +145,7 @@ export async function eachMessageWebhooksHandlers(
         // exit early if no webhooks nor resthooks
         return
     }
-    const event = convertToIngestionEvent(clickHouseEvent)
+    const event = convertToPostIngestionEvent(clickHouseEvent)
 
     // TODO: previously onEvent and Webhooks were executed in the same process,
     // and onEvent would call convertToProcessedPluginEvent, which ends up
```

Comment:
@benjackwhite  check out this comment tied to mutating the event or not - seems like we should always do this, so we can move that into `extendPostIngestionEventWithElementsList` instead

Reply:
I don't understand the comment or your comment. What is it your suggesting?

Reply:
I think I sort of got it... I kept it outside of that method so that we have explicit coverage of the reasoning behind it